Lesson-1/assignment/81_黄鑫实_第一课作业

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.4.14;
+
+contract payRollHomework {
+    uint constant payDuration = 10 seconds;
+
+    address owner;     //the employer's address
+       uint salary;    // employee's salary
+    address employee;  //employee's address
+       uint lastPayday;//the last date of payment
+
+    // constructor,once deploy, set owner as as the owner of contract
+    function payRollHomework() {
+        owner = msg.sender;
+        salary = 1;
+    }
+    
+    //udpate the address and salary to the other employee
+    function updateEmployee(address e, uint s) {
+        require(msg.sender == owner);
+        //all of empty addres start by 0x0
+        if(employee != 0x0) {
+            uint payment = salary * (now - lastPayday) / payDuration;
+            employee.transfer(payment);
+        }
```

Comment:
this code snippet: if(employee != 0x0) {
          uint payment = salary * (now - lastPayday) / payDuration;
            employee.transfer(payment);
        }

is for paying all the rest due, because the employer owes the employee the salary from the last pay day to now.